src/DroneSafetyApi/Controllers/HeatmapsController.cs

```diff
@@ -27,11 +32,18 @@ public IActionResult Get([FromQuery]HeatmapsQuery query)
             {
                 return new BadRequestResult();
             }
-            // TODO: Decide optimal radius value, and put value/computation in appropriate place.
-            var intersectionHazards = DataPoints.GetDataPointsInRadius(query.Centre, 100000);
-            var heatmapsResponse = DataPointsToHeatmaps.ConvertToHeatmapResponse(
-                query.Resolution,
+            var radius = (int) (Math.Sqrt(((query.Area.Max.Latitude - query.Area.Min.Latitude) *
+                (query.Area.Max.Latitude - query.Area.Min.Latitude)) +
+                ((query.Area.Max.Longitude - query.Area.Min.Longitude) *
+                (query.Area.Max.Longitude - query.Area.Min.Longitude)))) * MetresInLatDegree;
+            var intersectionHazards = ((IEnumerable<Hazard>)
+                Hazards.GetHazardsInRadius<CircularHazard>(query.Centre, radius, "Circle", query.Time))
```

Comment:
No because we are differentiating by data sources not shapes, and so we need to do it this way, Josh can explain it to you but this is the best way for us to do it atm with our current setup